src/functions/guild/getGuildTemplate.js

```diff
@@ -0,0 +1,19 @@
+module.exports = async (d) => {
+  const data = d.util.aoiFunc(d);
+  if (data.err) return d.error(data.err);
+  let template;
+
+  const [templateCode, property] = data.inside.splits;
+
+  try {
+    template = await d.client.fetchGuildTemplate(`https://discord.new/${templateCode}`);
+  } catch (err) {
+    return d.aoiError.fnError(d, "custom", { inside: data.inside }, "Template ID");
```

Comment:
this need better error message

Reply:
It does have a good error message no? `AoiError: Invalid Template ID [templateCode;property]`